src/Symfony/Component/Config/Resource/ReflectionClassResource.php

```diff
@@ -154,13 +154,15 @@ private function generateSignature(\ReflectionClass $class): iterable
                     continue;
                 }
 
-                if (!$p->isDefaultValueConstant() || \defined($p->getDefaultValueConstantName())) {
+                $defaultValueConstantName = preg_replace('/^parent::/', 'pimax\Messages\Message::', $p->getDefaultValueConstantName());
```

Comment:
Harcoding "pimax" in the source code of Symfony doesn't make sense.
Maybe you can provide us with a reproducer? Or better, a failing test case?

Reply:
Isn't that the name of the type of `$m`?

Reply:
I'm sorry I don't understand, there is no `$m` on this line.